card/lib/card/auth.rb

```diff
@@ -214,7 +213,9 @@ def always_ok_usr_id? usr_id
       def createable_types
         type_names = Auth.as_bot do
           Card.search type: Card::CardtypeID, return: :name,
-                      not: { codename: ['in'] + NON_CREATEABLE_TYPES }
+                      not: {
+                        codename: ['in'] + Card.config.non_createable_types
+                      }
```

Comment:
I'll merge this in for now, but I would suspect this fits more naturally as a class variable than a Card.config setting.  I would think that the rule of thumb should be that we only use Card.config if we really want people manually changing the setting.